ush/forecast_postdet.sh

```diff
@@ -304,6 +307,7 @@ EOF
   FNALBC=${FNALBC:-"${FIX_SFC}/${CASE}.snowfree_albedo.tileX.nc"}
   FNVETC=${FNVETC:-"${FIX_SFC}/${CASE}.vegetation_type.tileX.nc"}
   FNSOTC=${FNSOTC:-"${FIX_SFC}/${CASE}.soil_type.tileX.nc"}
+  FNSOCC=${FNSOCC:-"${FIX_SFC}/${CASE}.soil_color.tileX.nc"}
```

Comment:
@WalterKolczynski-NOAA  Is this file available for all resolutions from `C48` all the way up to `C768`?

Reply:
From C48 to C3072.